sotopia/ui/streamlit_ui/pages/add_scenarios.py

```diff
@@ -0,0 +1,123 @@
+"""
+Definition
+@app.post("/scenarios/", response_model=str)
+async def create_scenario(scenario: EnvironmentProfileWrapper) -> str:
+    scenario_profile = EnvironmentProfile(**scenario.model_dump())
+    scenario_profile.save()
+    pk = scenario_profile.pk
+    assert pk is not None
+    return pk
+
+class EnvironmentProfileWrapper(BaseModel):
+    Wrapper for EnvironmentProfile to avoid pydantic v2 issues
+
+    codename: str
+    source: str = ""
+    scenario: str = ""
+    agent_goals: list[str] = []
+    relationship: Literal[0, 1, 2, 3, 4, 5] = 0
+    age_constraint: str | None = None
+    occupation_constraint: str | None = None
+    agent_constraint: list[list[str]] | None = None
+    tag: str = ""
+
+class RelationshipType(IntEnum):
+    stranger = 0
+    know_by_name = 1
+    acquaintance = 2
+    friend = 3
+    romantic_relationship = 4
+    family_member = 5
+
+The age constraint of the environment, a list of tuples, each tuple is a range of age, e.g., '[(18, 25), (30, 40)]'
+means the environment is only available to agent one between 18 and 25, and agent two between 30 and 40
+"""
+
+import streamlit as st
+from sotopia.ui.fastapi_server import EnvironmentProfileWrapper
+from sotopia.database import RelationshipType
+import requests
+
+
+def local_css(file_name: str) -> None:
+    with open(file_name) as f:
+        st.markdown(f"<style>{f.read()}</style>", unsafe_allow_html=True)
+
+
+def rendering_scenario_form() -> None:
```

Comment:
add a default tag somewhere for now about user-created scenarios to avoid polluting the original scenarios

Reply:
same for characters